src/ISA/ISA/ARCTypes/CompositeHeader.fs

```diff
@@ -1,30 +1,177 @@
 ﻿namespace ISA
 
+open Fable.Core
+
+// TODO: Do we need FreeText of string on this one too? Do we want to allow other IOTypes?
+type IOType =
+    | Source
+    | Sample
+    | Data
+    | RawData
+    | ProcessedData
+    | Material
+
+    with
+
+    member this.asInput = $"Input [{this}]"
+    member this.asOutput = $"Output [{this}]"
+
+    /// Used to match exact string to IOType
```

Comment:
Should these parsing functions be part if ISA.Spreadsheet?

Reply:
I would only put the main conversion between our model and FsSpreadsheet model into ISA.Spreadsheet